diplomacy/web/.eslintrc.js

```diff
@@ -23,5 +23,6 @@
         "semi": [2, "always", { "omitLastInOneLineBlock": true}],
         "no-prototype-builtins": "off",
         "no-loss-of-precision": "off",
+        // "no-unused-vars": ["warn", { "ignoreRestSiblings": true }]
```

Comment:
Replaced json config with js config, so as to allow comments and more advanced options. I was going to tweak some eslint warnings for development- but left for another time as seen by the new, commented out, rule.